src/components/breadcrumbs/crumb/crumb.component.tsx

```diff
@@ -0,0 +1,33 @@
+import { LinkProps } from "components/link/link.component";
+import React from "react";
+import tagComponent, {
+  TagProps,
+} from "../../../__internal__/utils/helpers/tags";
+import StyledCrumb from "./crumb.style";
+import Divider from "./divider.component";
+
+export interface CrumbProps
+  extends Omit<LinkProps, "toolTipMessage" | "tooltipPosition">,
```

Comment:
We might need to omit the `icon` prop here as well as the DS spec says to not use them in Breadcrumbs